### ISSUE TITLE
New SDK Feature Class Symbology

### ISSUE THREAD

**Issue by kechinn** [WSDOT/HATS#1074]
Noticed that some of the symbols with text are showing up strangely (e.g. Other Category-Salt and Sand Storage Facilities and Liquid Deicer Tank). Does the new SDK have issues with that, or is there a way to adjust?

Also, any way to minimize the size of all the symbols so it's more like the old version? It's kinda nice that they are a bit bigger, but it seems like the current beta ones are too big when things get crowded, forcing user to zoom in a lot closer than before to see features underneath other features. Is there a middle ground, a happy medium haha?

**Comment by ReedHunter-Wsdot** [comment]
When you have a chance tomorrow morning or after, please redownload the offline features map and take a look at the new symbol sizes for those three layers and let me know which symbols sizes you'd prefer.  The new file will only have the changed symbols starting tomorrow morning.

I have adjusted the map service to change symbology for those three layers.  I suspect the new Sdk is doing a better job than the last one was at honoring the symbol sizes.  Other category salt, sand storage facilities, and liquid deicer tank were all 36 point sized symbols where as most other point symbols we use are at 24 points.    I have made Liquid Deicer Tank 30 points, and the other two 26 points each.
Automation will create a new offline features map early tomorrow morning after which we can take a look at the changes.  

In the past whenever I changed symbology on the map service it didn't break map sync compatibility with older offline map files.  We should then be free to adjust symbols like this until was have what we want without interfering with other testers.

**Comment by ReedHunter-Wsdot** [comment]
The new Sdk is not honoring the offsets of the letters for these symbols.  The 100.3 Sdk also doesn't let me reach into the guts of these symbols to fix the offset of the letters.

- I'll post the issue on Geonet so that it's on Esri's radar to get fixed in a future Sdk release.
- Until then I'll fix the symbols by removing the letters since they're not legible.  
- I'll also change the size of the symbols images that are loaded into the Toc since in that section the images are so big they're spilling over into the adjacent Toc cells.

**Comment by ReedHunter-Wsdot** [comment]
Toc updated so that oversized symbols are shrunk down to 28x28.

**Comment by ReedHunter-Wsdot** [comment]
The map service has been updated without the text in the symbols, so they look about as good as they can at the 100.3 Sdk. 

I have posted a tech support request with Esri showing them the problem.  I gave them the original symbol document showing how it should look, an extract of our data, and step by step instructions how they can modify one of their own developer samples to load up that data and see the problem in 100.3 for themselves.
